packages/api/AGENTS.md

```
@@ -1,3 +1,11 @@
+## Testing
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)